src/main/java/CommandErase.java

```diff
@@ -0,0 +1,22 @@
+public class CommandErase extends Command {
+    private int position;
+
+    public CommandErase(int position) {
+        this.position = position - 1;
```

Comment:
According the single responsibility principle, why not do the decrement when it's parsed.